core/src/main/scala/org/http4s/ContentCoding.scala

```diff
@@ -114,7 +115,8 @@ object ContentCoding {
   }
 
   implicit val http4sOrderForContentCoding: Order[ContentCoding] =
-    Order.by(c => (c.coding.toLowerCase, c.qValue))
+    Order.by(c => (c.coding.toLowerCase(ju.Locale.ENGLISH), c.qValue))
```

Comment:
Hmm. Maybe. `compareToIgnoreCase` is character-by-character, like `equalIgnoreCase`, so I'm having a hard time conceptualizing why that wouldn't work.  `toLowerCase` can change the lengths of the strings, and I fear is open to more corner cases.  And the default locale on my machine is `en_US` anyway.

If we're going to use `.toLowerCase`, I agree with this change, to not rely on the default locale.  It still doesn't feel right, but I don't have a counterexample.

Reply:
@rossabaker checkout https://github.com/http4s/http4s/issues/3328#issuecomment-645103412